### ISSUE TITLE
Platform is not attached to track error

### ISSUE THREAD

**Issue by sidshekar** [Seanba/SuperTiled2Unity#125]
Hi! I am not sure what I am doing wrong. The track and the platform gets imported but no matter what I always get the same error "platform is not attached to track error". I'm not sure if thins is because the points are floats and not ints as I am not what else could be causing it as I am using you example code as is. 

Also on a side note how would I make sure the distance is actually correct when I import the map. As my tilesize as 64 so I shouldnt be getting floats as distance anyway.

Thanks!! 

![image](https://user-images.githubusercontent.com/2862097/73699107-d03b0380-4747-11ea-9e3a-2e0d0473a7ce.png)

**Comment by sidshekar** [comment]
So I solved the issue in a different way. Anyway I was creating the moving platform path using the using the object layer instead of creating a separate map layer. This path get imported as a PolygonCollider2D. So in the code I had to replace EdgeCollider2D with PolygonCOllider2D. Then I import the track as usual, Replace the track with platform and destroy it after assigning the platform to a track.

![image](https://user-images.githubusercontent.com/2862097/73706710-01bec980-475e-11ea-9860-6c604b48948a.png)

Then in the movingPlatformOnTrack class I changed the AssignTrackIfClose function to take a PolygonCollider2D

and Since the points are local instead of world I added the object position to the local coords.

![image](https://user-images.githubusercontent.com/2862097/73706834-6ed25f00-475e-11ea-9d7f-2152483422a8.png)

I deleted the path from the Track Layer and just use the graphics. Also reduced the speed to 1.0f.

Now the path is closed and the platform follows the path. 

![platform](https://user-images.githubusercontent.com/2862097/73707039-1fd8f980-475f-11ea-85c1-4830dc8d0d5e.gif)
